### PR TITLE
Inclusive language

### PR DESCRIPTION
**Pull request by baptiste-b-pegasys**
I propose some changes on terms to be more inclusive, following this guideline: https://www.aswf.io/blog/inclusive-language/

The biggest impact is the `whitelist` flag that needs to be deprecated, then removed in a later version.

There are a lot of terms in `signer/fourbyte`, do you think it could be done in a later PR?

### CODE REVIEW

**Comment by ligi** [comment]
you often change "like crazy" to unpredictably where it is still predictable. Think the meaning is often more like "with high frequency"

**Comment by fjl** [comment]
Some of these changes cannot be made for backwards-compatibility reasons.

**Comment by gballet** [comment]
Thank you for your PR. Our policy is to strive to use a more inclusive language in our new PRs, while not accepting modification with a large footprint, that do not bring a technical fix and clobber the git history.

I shall also point out that "with a high frequency" does not correctly capture the meaning of "like crazy", as the former incorrectly conveys a meaning of regularity while the latter doesn't. The more correct concept would be "with high intensity". Also, "AllowList" is not grammatically correct: verbs in the infinitive can not function as adjectives. I suggest using the past participle "AllowedList".

**Comment by baptiste-b-pegasys** [comment]
appart the grammar, what is the suggestion? To split it into small PRs?
or these modifications are bad changes anyway, and it's definitely closed?

**Comment by holiman** [comment]
Yeah I think smaller PRs might be better. Some of them are internal renamings, other actually modify the API. 
I'm fine with following the guidelines from https://www.aswf.io/blog/inclusive-language/.

**Comment by holiman** [comment]
> The more correct concept would be "with high intensity".

And actually, I think neither "unpredictably" or "with high frequency" or "with high intensity" captures the meaning of "do X like crazy".

**Comment by karalabe** [comment]
I kind of agree with @gballet here.Clearly care should be taken to avoid potentially problematic phrasings, but I don't really see the point to go and change existing code just to replace wording where there was never any mal-intent.

**Comment by holiman** [comment]
I reviewed it a bit more thoroughly. I think many of the changes were good, even better than the original. Like, "// Don't scan the USB too frequently if the user fetches wallets in a loop" is IMO better than "// Don't scan the USB like crazy it the user fetches wallets in a loop". 

I'm thinking that abiding by a set of rules makes things easy.

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -234,8 +234,12 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
-	WhitelistFlag = cli.StringFlag{
+	DeprecatedAllowListFlag = cli.StringFlag{
 		Name:  "whitelist",
+		Usage: "[DEPRECATED: you shall replace it by 'allowlist'] Comma separated block number-to-hash mappings to enforce (<number>=<hash>)",
```
Comment (holiman) [inline cmd/utils/flags.go:239]: We actually have a system for deprecating flags, making them not appear in the normal listing, but in the extended help. 
Also, "you shall" sounds a bit ... dramatic?

File: cmd/utils/flags.go [hunk 2]
```diff
@@ -1403,26 +1407,29 @@ func setMiner(ctx *cli.Context, cfg *miner.Config) {
...
 		return
 	}
-	cfg.Whitelist = make(map[uint64]common.Hash)
-	for _, entry := range strings.Split(whitelist, ",") {
+	cfg.AllowList = make(map[uint64]common.Hash)
+	for _, entry := range strings.Split(allowList, ",") {
 		parts := strings.Split(entry, "=")
 		if len(parts) != 2 {
-			Fatalf("Invalid whitelist entry: %s", entry)
+			Fatalf("Invalid allow list entry: %s", entry)
```
Comment (holiman) [inline cmd/utils/flags.go:1422]: Seems like it should be one word, "allowlist"

File: core/tx_pool.go [hunk 3]
```diff
@@ -635,7 +635,7 @@ func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 // pending or queued one, it overwrites the previous transaction if its price is higher.
 //
 // If a newly added transaction is marked as local, its sending account will be
-// whitelisted, preventing any associated transaction from being dropped out of the pool
+// allowed, preventing any associated transaction from being dropped out of the pool
```
Comment (holiman) [inline core/tx_pool.go:638]: not "will be allowed", but "will be added to the allowlist"

File: eth/handler_eth.go [hunk 4]
```diff
@@ -140,13 +140,13 @@ func (h *ethHandler) handleHeaders(peer *eth.Peer, headers []*types.Header) erro
 			}
 			return nil
 		}
-		// Otherwise if it's a whitelisted block, validate against the set
-		if want, ok := h.whitelist[headers[0].Number.Uint64()]; ok {
+		// Otherwise if it's a allowed block, validate against the set
```
Comment (holiman) [inline eth/handler_eth.go:143]: All blocks are allowed, but only some of them are in the allowList.

File: node/rpcstack.go [hunk 5]
```diff
@@ -515,20 +515,20 @@ func (is *ipcServer) stop() error {
 	return err
 }
 
-// RegisterApisFromWhitelist checks the given modules' availability, generates a whitelist based on the allowed modules,
+// RegisterApisFromAllowList checks the given modules' availability, generates a allow List based on the allowed modules,
```
Comment (holiman) [inline node/rpcstack.go:518]: a**n** allowlist

File: p2p/dial.go [hunk 6]
```diff
@@ -402,7 +402,7 @@ func (d *dialScheduler) checkDial(n *enode.Node) error {
 		return errAlreadyConnected
 	}
 	if d.netRestrict != nil && !d.netRestrict.Contains(n.IP()) {
-		return errNotWhitelisted
+		return errNotAllowedListed
```
Comment (holiman) [inline p2p/dial.go:405]: NotAllowedListed? That's a double tempus form. `NotAllowlisted` maybe

File: p2p/msgrate/msgrate.go [hunk 7]
```diff
@@ -80,7 +80,7 @@ const rttMinConfidence = 0.1
 // behavior similar to our median peers.
 const ttlScaling = 3
 
-// ttlLimit is the maximum timeout allowance to prevent reaching crazy numbers
+// ttlLimit is the maximum timeout allowance to prevent reaching unexpected numbers
```
Comment (holiman) [inline p2p/msgrate/msgrate.go:83]: s/unexpected/out of bounds

**Review by holiman — COMMENTED** [review]
yeah I know it's closed, adding my 5c anyway

**Pull request closed without merging**